### PR TITLE
Adding code test gasused field [WIP]

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

File: bin/runWast.js [hunk 1]
```diff
@@ -11,11 +11,11 @@ const instance = Kernal.codeHandler(opsWasm)
 let sp = instance.exports.main()
```
Comment (kumavis) [inline bin/runWast.js]: what is `sp`?

File: index.js [hunk 2]
```diff
@@ -44,33 +46,62 @@ const interfaceImportMap = {
...
+// * the beginning for EVM code
+// * a GAS opcode (TODO)
+// * a JUMPDEST opcode
+// * After a JUMPI opcode
+// @param {Integer[]} evmCode the evm byte code
+// @param {Boolean} stackTrace if `true` generates a stack trace
+// @return {String}
 exports.compileEVM = function (evmCode, stackTrace) {
+  // this keep track of the opcode we have found so far. This will be used to
+  // to figure out what .wast files to include
```
Comment (kumavis) [inline index.js]: yay comments! lots of new comments : )

File: bin/runWast.js (hunk 1 above)
Comment (wanderer) [inline bin/runWast.js]: stack pointer

Comment (wanderer) [inline bin/runWast.js]: but main there doesn't return the sp so this code make no sense :-/

**Pull request merged**